build-aux/setcap-setuid.sh

```diff
@@ -15,9 +15,15 @@ _log() {
 
 case "$perm_type" in
 	caps)
-		params="cap_net_raw+p"
-		_log "calling: $setcap $params $exec_path"
-		"$setcap" $params "$exec_path"
+		if [ "$2" = "rdisc" ]; then
+			params="cap_net_raw,cap_net_admin+ep"
+			_log "calling: $setcap $params $exec_path"
+			"$setcap" $params "$exec_path"
+		else
+			params="cap_net_raw+p"
+			_log "calling: $setcap $params $exec_path"
+			"$setcap" $params "$exec_path"
+		fi
 	;;
```

Comment:
Why do you duplicate `_log` and `$setcap` call? Why not just:
```
params="cap_net_raw+p"
[ "$2" = "rdisc" ] && params="cap_net_raw,cap_net_admin+ep"
_log "calling: $setcap $params $exec_path"
"$setcap" $params "$exec_path"

```

Reply:
I also wonder if `cap_net_raw,cap_net_admin+p` would be enough.

Reply:
Yes, it must be +ep, not only +p or it will be error if you will try to run rdisc by unprivileged user